### DISCUSSION TITLE
blog/2022-in-review

### DISCUSSION THREAD

**Discussion by giscus** [TkDodo/blog-comments#90] [Announcements]
# blog/2022-in-review

A technical blog about frontend-development, TypeScript and React

https://tkdodo.eu/blog/2022-in-review

**Comment by Moshyfawn** [comment]
I like your attitude towards the future :D

Happy Holidays, Dominik! Thank you for the incredible content!

**Reply by TkDodo** [reply]
Thank you Oleg - happy new year to you too 🎉

**Comment by ardeora** [comment]
Wish you a very happy new year Dominik! 

Fun fact, I started reading your blogs this year. I have to admit, I have been awestruck by your ability to explain complex concepts in a way that really clicks for me. As someone who is interested in software development, I have found your articles to be incredibly valuable. They have given me a new perspective on how to approach my work, and I have found myself getting so excited as I read and learn from your insights.

Thank you for sharing your knowledge and experience with the rest of us. Your blog is a true treasure trove of information and inspiration, and I can't wait to see what you have in store for us next.

**Reply by TkDodo** [reply]
This is one of the nicest things someone has ever said to me ❤️. We're so lucky to have you on board as a solid-query maintainer. Happy new year to you 🎉

**Comment by RichEwin** [comment]
I really enjoy reading your blog posts, you explain technical concepts so well. I lead the project at my workplace to migrate from redux to RQ with success and you helped me overcome some challenges which I am so grateful for. 

The high level of work you output is something I inspire to. It is my goal to contribute to an open source library one day.

Keep up the good work and I hope one day we we can work together 🚀

**Reply by TkDodo** [reply]
Thank you Richard - that's a great success story 🎉

**Comment by papadavis47** [comment]
I did learn a lot from your appearance on Learn With Jason and I am happy to hear you will be producing more video content. I have appreciated your blog posts since I first came across them some months ago. Your content is very helpful. Thank you.
